src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -249,6 +249,12 @@ pscanalpha.usernameidor.otherinfo = The hash was an {0}, with value: {1}
 pscanalpha.usernameidor.soln = Use per user or session indirect object references (create a temporary mapping at time of use). Or, ensure that each use of a direct object reference is tied to an authorization check to ensure the user is authorized for the requested object. 
 pscanalpha.usernameidor.refs = https://www.owasp.org/index.php/Top_10_2013-A4-Insecure_Direct_Object_References\nhttps://www.owasp.org/index.php/Testing_for_Insecure_Direct_Object_References_(OTG-AUTHZ-004)
 
+pscanrules.xaspnetversioncanner.name = X-AspNet-Version Response Header Scanner
+pscanrules.xaspnetversioncanner.desc =  Server Leaks Information via "X-AspNet-Version"/"X-AspNetMvc-Version" HTTP Response Header Field(s).
```

Comment:
This is good.

Think there's any value in mentioning wasted bandwidth? (Times 10s of thousands of responses, it adds up.)

Reply:
Yes, I can add it - but this is also true for `X-Powered-By` and `Server` header, what do you think of adding it to all of them?

Reply:
Don't worry about it, leave as is.